<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mate_fonc</class>
 <widget class="QDialog" name="mate_fonc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1441</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(217, 217, 217);</string>
  </property>
  <widget class="QPushButton" name="accueil">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(36, 179, 122);
font: 900 12pt &quot;Franklin Gothic&quot;;
color: rgb(0, 85, 0);</string>
   </property>
   <property name="text">
    <string>Accueil</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-40</y>
     <width>241</width>
     <height>671</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(36, 179, 122);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QPushButton" name="ges_emp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 179, 122);
color:rgb(0, 125, 209);
font: 900 12pt &quot;Franklin Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Gestion Employé</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ges_mat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(211, 211, 211);
color: rgb(255, 191, 62);
font: 900 12pt &quot;Franklin Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Gestion Matériels</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ges_es_vert">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 179, 122);
color:rgb(0, 125, 209);
font: 900 12pt &quot;Franklin Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Gestion Espaces Verts</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ges_vehic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 179, 122);
color:rgb(0, 125, 209);
font: 900 12pt &quot;Franklin Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Gestion Véhicule</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ges_appar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 179, 122);
color:rgb(0, 125, 209);
font: 900 12pt &quot;Franklin Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Gestion Appartements</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deconnect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 179, 122);
color: rgb(170, 0, 0);
font: 900 12pt &quot;Franklin Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Se Déconnecter</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>59</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="fonction">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>171</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(36, 179, 122);
font: 900 12pt &quot;Franklin Gothic&quot;;
color: rgb(0, 85, 0);</string>
   </property>
   <property name="text">
    <string>Fonctionalités</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>1171</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox
{
background-color: rgb(0, 125, 209);
color:  rgb(255, 191, 62);
font: 900 12pt &quot;Franklin Gothic&quot;;
border: 2px solid ;
border-radius: 12px; 
padding: 5px;
border-color: rgb(255, 191, 62);
}</string>
   </property>
   <property name="title">
    <string>Notification</string>
   </property>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>541</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>991</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 6pt &quot;Segoe UI&quot;;
font: 7pt &quot;Segoe UI&quot;;
font: 900 9pt &quot;Franklin Gothic&quot;;
color: rgb(0, 0, 0);
</string>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>157</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Date</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <strikeout>false</strikeout>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Notifications</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>1151</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>Statestique</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>240</y>
       <width>31</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </widget>
    <widget class="QLabel" name="stat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>1131</width>
       <height>241</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="supprimer2">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>120</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 179, 122);
font: 900 12pt &quot;Franklin Gothic&quot;;
color: rgb(0, 85, 0);</string>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
